perceval/runtime/job_group.py

```diff
@@ -222,9 +231,108 @@ def progress(self) -> list:
 
         self._write_job_group_to_disk()  # rewrites the group information on disk
 
-        # replace None with 'not_sent' - more readable
-        status_list = [status if status is not None else "not_sent" for status in status_jobs_in_group]
-        return status_list
+        return status_jobs_in_group
+
+    @staticmethod
+    def _map_job_status_category(status_entry: str) -> JobCategories:
```

Comment:
Move this to the JobCategories class

Reply:
Following our discussions, removed the enum class `JobCategories`.